package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Data typing discovery and MIME content-type handling across all media types that the Scratch environment expects",
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/LLK/scratch-asset-types.git"
+    "url": "git+ssh://git@github.com/scratchfoundation/scratch-asset-types.git"
   },
   "scripts": {
-    "test": "make test"
+    "test": "tap ./test/unit/*.js",
```

Comment:
Hmm... it's kind of awkward to have this out of sync with what the `Makefile` does. It doesn't currently matter because `./test/functional/` and `./test/integration/` don't exist, but it seems like it's setting the stage for a mistake in the future. I'd suggest either migrating everything out of `Makefile` and deleting it, or changing the `test` script back to `make test`.